addons/sale/models/sale_order_line.py

```diff
@@ -313,13 +313,19 @@ def _compute_name(self):
                 line = line.with_context(lang=lang)
             name = line._get_sale_order_line_multiline_description_sale()
             if line.is_downpayment and not line.display_type:
-                context = {'lang': lang}
```

Comment:
why do you remove the context here?
It's used by the translation api

Reply:
I removed it originally because I saw that it wasn't used in saas-17.1. 
I restored it to its previous in the recent commit since its used by the translation api.